### ISSUE TITLE
juvenile granulosa cell tumour testis

### ISSUE THREAD

**Issue by TonyRoscioli** [obophenotype/human-phenotype-ontology#7453]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Juvenile type granulosa cell tumor

NCIT:C4207

-- testicular
Juvenile type testicular granulosa cell tumor
Def: Juvenile granulosa cell tumor of the testis of neonates and infants is an uncommon lesion frequently associated with abnormal sex chromosome and ambiguous genitalia.
PMID:17652550
NCIT:C4207

--ovarian
Juvenile granulosa cell tumors are rare ovarian neoplasms diagnosed in the pediatric population. These tumors are commonly clinically significant due to their hormone secreting properties, resulting in symptoms of [precocious puberty. While primary imaging characteristics are non–specific ranging from cystic to solid masses, secondary characteristics such as uterine enlargement, endometrial thickening, and breast bud development abnormal for age can be helpful in diagnosis.
PMID:34285730, NCIT:C7289

**Comment by pnrobinson** [comment]
added as HP_0034380
